src/lib/modernExtend.ts

```diff
@@ -754,6 +754,101 @@ export function batteryPercentage(args?: Partial<NumericArgs>) {
     });
 }
 
+export interface BatteryArgs {
+    voltageToPercentage?: string | {min: number, max: number}, dontDividePercentage?: boolean,
```

Comment:
I see a `voltageToPercentage` option, but there is no place in this function where voltage reporting is setup.

Reply:
Done